pages/finance.tsx

```diff
@@ -1,13 +1,25 @@
 import { useUser } from '@auth0/nextjs-auth0'
-import React from 'react'
+import { Button, makeStyles } from '@material-ui/core'
+import React, { useState } from 'react'
 
 import BaseLayout from '../components/common/BaseLayout'
 import Loader from '../components/common/Loader'
 import FinanceDateHeader from '../components/finance/FinanceDateHeader'
-import { dateAfterDays, dateBeforeDays } from '../lib/date-helper'
+import { dateAfterDays, dateAfterDaysFromToday, dateBeforeDays, dateBeforeDaysFromToday } from '../lib/date-helper'
+
+const useStyles = makeStyles({
+  button: {
+    marginLeft: '1rem',
+  },
+})
 
 function Finance() {
+  const classes = useStyles()
   const { user, isLoading, error } = useUser()
+  const [values, setValues] = useState({
+    startDate: dateBeforeDaysFromToday(3),
```

Comment:
do not hardcode 3 days. Find out how many days will fit in the given width

Reply:
planning to do as seperate issue

Reply:
abstracted it out as dateHeaderCount though.